Patches/[O21] Outer Rim Galaxies/Mandalore/Patch_OuterRim_Mandalore_Apparel.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	
+	<Operation Class="PatchOperationFindMod">
+    <mods>
+		<li>[O21] Outer Rim Galaxies [DevBuild]</li>
+    </mods>
+		<match Class="PatchOperationSequence">
+		<operations>
+		
+			<li Class="PatchOperationConditional">
+				<xpath>Defs/ThingDef[defName="O21_OR_AmbanSniperRifle"]</xpath>
+				<match Class="PatchOperationSequence">
+					<operations>
+					
+					<li Class="PatchOperationReplace">
+						<xpath>Defs/ThingDef[defName="O21_Mando_BasicArmor"]/statBases/ArmorRating_Sharp</xpath>
```

Comment:
This became, essentially, `O21_OR_MandalorianChest_Simple`. There was also a helmet and set of pauldrons added.

Reply:
Done